### ISSUE TITLE
Open hyperlinks from agenda buffer

### ISSUE THREAD

**Issue by stinovlas** [nvim-orgmode/orgmode#501]
### Does this feature exist in Emacs orgmode core?

N/A

### Feature value

You can already open hyperlinks in browser using `<Leader>oo` in org files. I'd find it useful to be able to open hyperlinks from agenda buffer as well. `<Leader>oo` is not used in agenda buffer, so I suggest we use it in here as well for the consistency reasons.

**Comment by jgollenz** [comment]
I started [working](https://github.com/nvim-orgmode/orgmode/pull/361#issue-1292435794) a couple of months ago. Feel free to run with it :+1:

**Comment by Maltimore** [comment]
Hi @stinovlas, I believe this feature was just implemented in de701e84ca7a73a0a389f6e6e25a8e278f12413b, is that correct? (cc @kristijanhusak )

**Comment by kristijanhusak** [comment]
That's correct @Maltimore , thanks for mentioning it. Closing this as resolved.
